playground/index.html

```diff
@@ -196,14 +197,14 @@
   const editor = ace.edit("editor")
   editor.setTheme("ace/theme/github_light_default");
   editor.setOptions({
-    // fontFamily: "Iosevka Web",
+    fontFamily: "Iosevka Web",
```

Comment:
I wrote an explanation in commit that commented these out: https://github.com/skvadrik/re2c/commit/885b665cfc109a7d0d4b7dcb1ad712298fcf1232:
```
Although Iosevka is a nice font, it may be a bit hard to read as it's very
narrow. It is better to use default system or browser monospace font, which
should be familiar to the user (and this is what we do for the examples on
re2c.org anyway).
```

I didn't delete them in case we change our mind later. If you don't like to leave them as comments, feel free to remove.

Reply:
> Although Iosevka is a nice font, it may be a bit hard to read as it's very
narrow.

You haven't seen [Quinze font](https://www.programmingfonts.org/#quinze) :)
But fair enough. I have removed Iosevka. The default font is good enough.